<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_config</class>
 <widget class="QDialog" name="chat_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create/Edit Chat</string>
  </property>
  <widget class="QGroupBox" name="gb_llm_config_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>LLM Configs List</string>
   </property>
   <widget class="QLabel" name="label_llm_config_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected llm config file path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_llm_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select LLM Configs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_llm_config">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit LLM Config</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_agents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>561</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Agents</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_agent">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_agent_llm_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_llm_config">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LLM Config</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_llm_config">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_advanced_llm_config">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_agent_human_input_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_human_input_mode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Human Input Mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_human_input_mode">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NEVER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALWAYS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TERMINATE</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_agent_system_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_system_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>System Message</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pte_system_message">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Agent System Message</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_code_execution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_code_execution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code Execution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_code_execution">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>LocalCommandLineCodeExecutor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">DockerCommandLineCodeExecutor</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="le_code_execution">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Working directory</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_tools">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_tools">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tools</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_tools">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_tool">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat</string>
   </property>
   <widget class="QFrame" name="frame_initiator_agent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_initiate_agent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Initiator agent</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_initiator_agent">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_recipient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_recipient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Recipient</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_recipient">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>GroupChatManager</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_max_turns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_max_turns">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Max turns</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_max_turns">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_summary_method">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_summary_method">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Summary Method</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_summary_method">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>last_msg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflection_with_llm</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_initial_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_initial_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial Message</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Chat Initial Message</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save_config">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>850</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_config">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>850</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chat_config_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>850</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected chat config file path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
